File: SCALA/Cap03/c03ex02.scala
// c03ex02.scala

object Main {
  def main(args: Array[String]): Unit = {
    var R, A: Double = 0.0

    print("Entre valor do raio: ")
    R = scala.io.StdIn.readDouble()

    A = Math.PI * Math.pow(R, 2)

    printf("Resultado = %.2f\n", A)

    print("\nTecle <Enter> para encerrar... ")
    scala.io.StdIn.readLine()
  }
}

File: DART/Cap03/c03ex02.dart
// c03ex02.dart

import 'dart:io';
import 'dart:math';

void main() {
  double? r;
  double? a;

  stdout.write("Entre valor do raio: ");
  r = double.tryParse(stdin.readLineSync() ?? '');

  a = pi * pow(r!, 2);

  print("Resultado = ${a.toStringAsFixed(2)}");

  stdout.write("\nTecle <Enter> para encerrar... ");
  stdin.readLineSync();
}
